tensorflow/core/kernels/dml_space_to_batch_op.cc

```diff
@@ -431,8 +431,9 @@ class DmlSpaceToBatchKernel : public DmlKernel {
       DmlKernelWrapper<DmlSpaceToBatchKernel<SpaceToBatchInitHelper>,   \
                        SpaceToBatchShapeHelper>);
 
-TF_CALL_float(REGISTER_DML_KERNEL);
-TF_CALL_half(REGISTER_DML_KERNEL);
+// TODO: re-register these kernels when we can depend on DML 1.15.
```

Comment:
Can we add a bug number for this to make sure it doesn't fall through the cracks?